Makefile.include

```diff
@@ -112,7 +112,9 @@ CFLAGS += -DCPU_$(CPUDEF)=\"$(CPU)\" -DRIOT_CPU=CPU_$(CPUDEF)
 CFLAGS += -DMCU_$(MCUDEF)=\"$(MCU)\" -DRIOT_MCU=MCU_$(MCUDEF)
 
 # OSX fails to create empty archives. Provide a wrapper to catch that error.
-ifneq (0, $(shell mkdir -p $(BINDIR); $(AR) -rc $(BINDIR)empty-archive.a 2> /dev/null; echo $$?))
+# llvm-ar does not understand `-rc`, but works for creating empty archives when
+# called with `rc` (without the leading dash).
```

Comment:
The comment is slightly superfluous as long as the commit message explains this. If you think it should be manifested somewhere, put it besides the `ARFLAGS` definition.